Content/Packets/GlassweaverWaitingPacket.cs

```diff
@@ -0,0 +1,48 @@
+﻿using NetEasy;
+using StarlightRiver.Content.Abilities;
+using StarlightRiver.Content.Bosses.GlassMiniboss;
+using StarlightRiver.Content.Bosses.VitricBoss;
+using StarlightRiver.Content.Dusts;
+using StarlightRiver.Core.Systems.CameraSystem;
+using System;
+using Terraria.ID;
+using static StarlightRiver.Content.Bosses.VitricBoss.VitricBoss;
+using static Terraria.ModLoader.ModContent;
+
+namespace StarlightRiver.Content.Packets
```

Comment:
I'm a tad confused on what this packet is doing VS send/recieve extra data in the glassweaver's ModNPC? Perhaps some clarification on how it differs would be valuable?